src/Microsoft.AspNetCore.Identity.UI/ConfigureApplicationCookieAreaSetup.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Identity
+{
+    internal class ConfigureApplicationCookieAreaSetup : IConfigureNamedOptions<CookieAuthenticationOptions>
```

Comment:
this can probably just be:

```
   class ConfigureApplicationCookieArea : ConfigureNamedOptions<CookieAuthenticationOptions> {
        public ConfigureApplicationCookieArea() : base(IdentityConstants.ApplicationScheme, o => {
                options.LoginPath = "/Identity/Account/Login";
                options.LogoutPath = "/Identity/Account/Logout"                
                options.AccessDeniedPath = "/Identity/Account/AccessDenied";
         }
     }
   }
````